benches/languages.rs

```diff
@@ -286,15 +284,15 @@ fn benchmark_configs(
 
     for (name, transform_config) in configs.into_iter() {
         group.throughput(Throughput::Elements(num_lines as u64));
-        group.bench_function(name.clone(), |b| {
+        group.bench_function(name, |b| {
             b.iter_batched(
                 || {
                     let mut config = source_config.clone();
-                    config.push_str(&transform_config);
+                    config.push_str(transform_config);
                     config.push_str(&sink_config);
 
                     let config = config::load_from_str(&config, config::Format::Toml)
-                        .expect(&format!("invalid TOML configuration: {}", &config));
+                        .unwrap_or_else(|_| panic!("invalid TOML configuration: {}", &config));
```

Comment:
I don't particularly like this change but if this was done with clippy --fix it's fine